statescript/executor.go

```diff
@@ -165,6 +176,28 @@ func execute(name string, timeout time.Duration) int {
 		return retCode(err)
 	}
 
+	var bts []byte
+	if stderr != nil {
+		bts, err = ioutil.ReadAll(stderr)
+		if err != nil {
+			log.Error(err)
+		}
+	}
+
+	if len(bts) > 0 {
+		if len(bts) > 10*1024 { // do not log more than 10KB at a time
+			buf := make([]byte, 10*1024)
+			n := copy(buf, bts)
```

Comment:
Not sure why do we need a copy of the slice here. We can easily get rid of it and use "slice" of the original slice. See this for reference: https://blog.golang.org/go-slices-usage-and-internals.
You can simply do something like this below `log.Errorf("stderr collected while running script %s [%s] (Truncated to 10KB)", name, string(bts[:10240]))`. I would simply add new variable `maxSize := 10*1024` and then use it here. I think you could also try to limit the number in `Errorf` statement using `%.10240s`. In C you can also use something like `printf("%.*s", len, string_to_print)`, but not sure if this works in Go.